interfaces/Config/templates/config_server.tmpl

```diff
@@ -223,6 +234,19 @@
                             <label class="config" for="displayname$cur">$T('srv-displayname')</label>
                             <input type="text" name="displayname" id="displayname$cur" value="$server['displayname']" />
                         </div>
+                        <div class="field-pair advanced-settings">
+                            <label class="config" for="expire_date">$T('srv-expire_date')</label>
+                            <input type="text" name="expire_date" id="expire_date$cur"  value="$server['expire_date']" />
```

Comment:
Let's use `type=date`.

Reply:
Well, blush, I wasn't even aware of that type.